qhub_ops/cli/render.py

```diff
@@ -13,6 +14,7 @@ def create_render_subcommand(subparser):
 
 
 def handle_render(args):
+    verify(args.config)
```

Comment:
`args.config` is the path to the configuration file. Does pydantic handle this? I thought it expected a python object dict/list etc. .

Reply:
I think adding should make it work

```
config_filename = pathlib.Path(args.config) 
 if not config_filename.is_file(): 
     raise ValueError( 
         f"passed in configuration filename={config_filename} must exist" 
     ) 
  
 with config_filename.open() as f: 
     config = yaml.safe_load(f.read()) 
```

Reply:
makes sense. i wasnt sure what was coming the config object.